src/core/public/chrome/ui/header/collapsible_nav_group_enabled_top.tsx

```diff
@@ -148,10 +149,10 @@ export const CollapsibleNavTop = ({
           />
         </EuiFlexItem>
       </EuiFlexGroup>
-      {currentNavGroup?.title && (
+      {currentNavGroup?.title && collapsibleNavHeaderRender && (
```

Comment:
I'm wondering why it checks `currentNavGroup?.title`?

Reply:
We use it for the case when workspace is not enabled, please check https://github.com/opensearch-project/OpenSearch-Dashboards/pull/8364#discussion_r1780662229

Reply:
When workspace is not enabled, `registerCollapsibleNavHeader` will not called, so `collapsibleNavHeaderRender` doesn't exist.

Reply:
ok I see